lab_8/task_8_1.py

```diff
@@ -0,0 +1,13 @@
+def users(users_list):
+    if not users_list:
+        print('We need to find some users!')
+    else:
+        for user in users_list:
+            welcome_text = 'Hello Admin, I hope you\'re well.' if user == 'Admin' else \
+                f'Hello {user} thank you for logging in again.'
+            print(welcome_text)
```

Comment:
do you remember, I'm all the time advising to put preferable case to if block and unwanted to else block. so, change it